pkg/accumulator/accumulator.go

```diff
@@ -0,0 +1,256 @@
+package accumulator
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"sniffer/pkg/config"
+	"sniffer/pkg/ebpfeng"
+	evData "sniffer/pkg/ebpfev/v1"
+
+	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/go-logger/helpers"
+)
+
+const (
+	ACCUMULATOR_SIZE = 10
+)
+
+const (
+	DROP_EVENT_OCCURRED = "drop event occurred\n"
+)
+
+type containersAccumulator struct {
+	accumulatorDataPerContainer map[string]chan evData.EventData
```

Comment:
the name of this map is not clear -  I assume the key is the containerID?
if so, maybe a better name is containerIdToAccumuationData?  or xxxToYYY ...